runtime/common/src/paras_registrar.rs

```diff
@@ -188,66 +201,91 @@ decl_module! {
 		#[weight = 0]
 		fn swap(origin, other: ParaId) {
```

Comment:
sidenote: what if a parachain wants to swap, but the other never swap in return, should there be a mecanism to unrequest swap. Or this is consider never to happen because parachain should have trusted each over with other mecanism.

Reply:
to "remove swap" you can claim a swap with yourself right?

Reply:
ah true